src/data/logData/logDataHooks.ts

```diff
@@ -117,6 +117,42 @@ export const useTraceIdLogs = (traceId: string): LogWithSource[] =>
     ),
   );
 
+// counts string "<unacked>|<acked>|<total>" (each is a number)
+export type MatchedCountsString = `${number}|${number}|${number}`;
+
+export const useMatchedAckedUnackedCount = (filter: string): MatchedCountsString =>
+  useSelector(
+    createSelector(
+      [
+        (state: RootState) => state.logData.logs,
+      ],
+      (logs) => countMatched(filter, logs),
+    ),
+  );
+
+function countMatched(filter: string, logs: Log[]): MatchedCountsString {
+  let unacked: number = 0;
+  let acked: number = 0;
+  let total: number = 0;
+  try {
+    const matchingRegex = RegExp(filter);
+    for (const l of logs) {
+      if (null === matchingRegex.exec(l.line)) {
```

Comment:
this is a place I introduced a bug, by checking only the match onto a single source of that line. Thankfully, was caught by an unrelated test because I started to disable the "match" button on no match 😅 😅